commands/plugin.go

```diff
@@ -104,9 +104,12 @@ func pluginAddCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		} else {
 			_, _, err = c.UploadPlugin(fileReader)
 		}
-
+		
+		var result error
 		if err != nil {
 			printer.PrintError("Unable to add plugin: " + args[i] + ". Error: " + err.Error())
+			result = multierror.Append(result, err)
+			return result
```

Comment:
We should continue to the loop here.

Reply:
Nope, we return in the first error with your change. That's something we don't want to do. We need to continue to iterate over the slice and get the errors at last.